src/components/PolicyTargetRefPage.tsx

```
@@ -0,0 +1,179 @@
+            );
+          default:
+            return null;
+        }
+      })}
+    </>
+  );
+
+  const renderBody = () => {
+    if (!policyLoaded) return <div>{t('Loading...')}</div>;
+    if (policyLoadError)
+      return <div>{t('Error loading policy: {{message}}', { message: policyLoadError.message })}</div>;
```

ESLint

[error] Replace `<div>{t('Error·loading·policy:·{{message}}',·{·message:·policyLoadError.message·})}</div>` with `(⏎········<div>{t('Error·loading·policy:·{{message}}',·{·message:·policyLoadError.message·})}</div>⏎······)`

(prettier/prettier)